packages/web/src/lib/environment.ts

```diff
@@ -6,3 +6,7 @@ export const ZOEKT_WEBSERVER_URL = getEnv(process.env.ZOEKT_WEBSERVER_URL, "http
 export const SHARD_MAX_MATCH_COUNT = getEnvNumber(process.env.SHARD_MAX_MATCH_COUNT, 10000);
 export const TOTAL_MAX_MATCH_COUNT = getEnvNumber(process.env.TOTAL_MAX_MATCH_COUNT, 100000);
 export const NODE_ENV = process.env.NODE_ENV;
+
+export const AUTH_JS_SECRET = getEnv(process.env.AUTH_JS_SECRET); // Generate using `npx auth secret`
```

Comment:
nit: why not use the AUTH_SECRET convention? https://authjs.dev/reference/core/errors#missingsecret